lib/src/article.g.dart

```diff
@@ -333,63 +333,93 @@ class ArticleBuilder implements Builder<Article, ArticleBuilder> {
   _$Article _$v;
 
   int _id;
+
```

Comment:
There changes in these files have nothing to do with the change being made. It's no big deal but, generally, it will be cleaner if the change is scoped.

Reply:
fixed this too.